app/Http/routes.php

```diff
@@ -168,20 +154,90 @@
         return view('dashboard.pages.edit_episode');
     });
 
-    Route::get('/channels', function () {
-        return view('dashboard.pages.view_channels');
-    });
-
-    Route::get('/channel/create', function () {
-        return view('dashboard.pages.create_channel');
-    });
+/*
+/-------------------------------------------------------------------------------
+/ Admin User
+/-------------------------------------------------------------------------------
+*/
+    Route::get('/users', [
+        'uses' => 'UserController@index',
+        'as'   => '/users'
+    ]);
+
+    Route::get('/user/edit/{id}', [
+        'uses' => 'UserController@editView',
+        'as'   => '/user/edit/{id}'
```

Comment:
**Code Quality - Nitpick*

The reason why you use named routes is to avoid having to reference urls again. So get nice subtle names for all the routes in this file instead of using the real routes as named routes. 

Example the named route here could be `'as' => 'user-edit'`